examples/rust/examples/maker/maker.rs

```diff
@@ -31,7 +33,8 @@ const SESSION_COOKIE_KEY: &str = "set-cookie";
 /// The Market Maker will receive Request For Quote (RFQ) from the Valorem server formatted as
 /// `QuoteRequest` and the MM needs to respond with `QuoteResponse`.
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
+async fn main() -> Result<(), anyhow::Error> {
+    // TODO(DRY)
```

Comment:
Not sure where the repeat is? If its in reference to the `run` calls, this repeat is required as the provider trait is not object safe. We can wrap it in a enum, however not sure if that would really make anything better.